x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/formatter/TextFormat.java

```diff
@@ -291,19 +290,10 @@ public Iterator<CheckedConsumer<Writer, IOException>> format(RestRequest request
             hasHeader(request) && esqlResponse.columns() != null
                 ? Iterators.single(writer -> row(writer, esqlResponse.columns().iterator(), ColumnInfo::name, delimiter))
                 : Collections.emptyIterator(),
-            Iterators.map(esqlResponse.values(), row -> writer -> row(writer, row, TextFormat::formatEsqlResultObject, delimiter))
+            Iterators.map(esqlResponse.values(), row -> writer -> row(writer, row, f -> Objects.toString(f, StringUtils.EMPTY), delimiter))
         );
     }
 
-    private static String formatEsqlResultObject(Object obj) {
-        // TODO: It would be nicer to override GeoPoint.toString() but that has consequences
-        if (obj instanceof SpatialPoint point) {
-            // SpatialPoint.toWKT maintains max precision, while ´%f´ conveniently simplifies for display purposes
-            return String.format(Locale.ROOT, "POINT (%f %f)", point.getX(), point.getY());
```

Comment:
There is similar code in TextFormatter, I wonder if that can be removed similarly?

Reply:
Done from TextFormatter too!